test/unit/mdc-auto-init/mdc-auto-init.test.js

```diff
@@ -18,14 +18,29 @@ import bel from 'bel';
 import td from 'testdouble';
 import {assert} from 'chai';
 import mdcAutoInit from '../../../packages/mdc-auto-init';
+import {MDCComponent} from '../../../packages/mdc-base';
 
-class FakeComponent {
+class FakeComponent extends MDCComponent {
   static attachTo(node) {
     return new this(node);
   }
 
   constructor(node) {
-    this.node = node;
+    super();
+    this.root_ = node;
+  }
+
+  get root() {
+    return this.root_;
+  }
+
+  getDefaultFoundation() {
```

Comment:
Some tests don't pass. The error is `Error: Subclasses must override getDefaultFoundation to return a properly configured foundation class`